observability/kepler/otelcol-config.yaml

```diff
@@ -0,0 +1,57 @@
+---
+extensions:
+  file_storage:
+    # this dir must exist on filesystem where otelcollector is running
+    directory: /etc/otelcol-logs
```

Comment:
Isnt there a better place for cached data? AFAIK `/var` should be mutable? Maybe we can use something like `/var/cache/opentelemetry-collector/`? wdyt?

Reply:
Yes lets treat /usr and /etc as read-only from apps point of view.